### DISCUSSION TITLE
Build output paths

### DISCUSSION THREAD

**Discussion by SoffiMeower** [obhq/obliteration#9] [General]
@ultimaweapon 

Hey there! Do you know the output folders for both Windows and Linux compilations?

**Comment by ultimaweapon** [comment]
It will be `build/src` if follow the instructions from README.

**Reply by SoffiMeower** [reply]
> It will be `build/src` if follow the instructions from README.

Does not seem to be that case for Ubuntu.

**Reply by SoffiMeower** [reply]
@ultimaweapon Yeah, Ubuntu builds do not output to build/src it seems...

**Reply by ultimaweapon** [reply]
Should not be possible. When you run `cmake -B build -D CMAKE_BUILD_TYPE=Release` it will create `build` directory and then you run `cmake --build build` it will produce `build/src`.

**Reply by SoffiMeower** [reply]
> Should not be possible. When you run `cmake -B build -D CMAKE_BUILD_TYPE=Release` it will create `build` directory and then you run `cmake --build build` it will produce `build/src`.

Well... Linux:
![image](https://user-images.githubusercontent.com/45863583/194445181-1eaae145-8634-46bc-94f5-f1d1a964d4e9.png)

And Windows:
![image](https://user-images.githubusercontent.com/45863583/194445241-c7c232e5-4574-466e-8747-cd5d56649307.png)

**Reply by ultimaweapon** [reply]
What is the directory structure that show from `ls`?
